### PR TITLE
OPENNLP-1360: Updating the Get Involved and Mailing List pages

### PR DESCRIPTION
**Pull request by jzonthemtn**
Formatting the mailing lists, removing list of wanted contributions, updating contribution steps.

### CODE REVIEW

File: src/main/jbake/content/get-involved.ad [hunk 1]
```diff
@@ -22,18 +22,47 @@
...
+
+[cols="3,3,3,3,3"]
+|===
+|Mailing List |Purpose |Subscribe |Unsubscribe |Archives
+
+|Users Mailing List
+|This list is the general users mailing list and is for posting most, if not all, relevant questions. If you are trying to decide which list, opennlp-dev or opennlp-users, post here first. Someone will kindly suggest you post to opennlp-dev if it is more appropriate there.
+|mailto:users-subscribe@users.apache.org[Subscribe]
+|mailto:users-unsubscribe@users.apache.org[Unsubscribe]
+|http://mail-archives.apache.org/mod_mbox/opennlp-users/[Archives]
```
Comment (kinow) [inline src/main/jbake/content/get-involved.ad]: https://lists.apache.org/list.html?users@opennlp.apache.org ?

It's redirected there when I open it, and some browsers/companies have started blocking http links too.

File: src/main/jbake/content/get-involved.ad [hunk 2]
```diff
@@ -22,18 +22,47 @@
...
 
-Before you start to contribute we suggest to join the user and development mailing lists.
+|Developers Mailing List
+|This list is for development discussions, patch suggestions, and current issues posted
+to the issue tracker for the project.
+|mailto:dev-subscribe@users.apache.org[Subscribe]
+|mailto:dev-unsubscribe@users.apache.org[Unsubscribe]
+|http://mail-archives.apache.org/mod_mbox/opennlp-dev/[Archives]
```
Comment (kinow) [inline src/main/jbake/content/get-involved.ad]: Ditto link above.

File: src/main/jbake/content/get-involved.ad [hunk 3]
```diff
@@ -22,18 +22,47 @@
...
-A contribution is always done through https://issues.apache.org/jira/browse/OPENNLP[Apache OpenNLP JIRA].
-Please https://issues.apache.org/jira/secure/CreateIssue.jspa?pid=12311215&issuetype=4[create a JIRA issue]
-which describes the change and attach a patch to it.
-Every received patch will be reviewed and then either be committed or we give appropriate feedback
-to change it.
+|Commits Mailing List
+|This list follows changes to the project. This list is _not_ for questions. This list is used by svn to post changes to the project. If you have questions about any content here, post them to the opennlp-dev list above.
+|mailto:commits-subscribe@users.apache.org[Subscribe]
+|mailto:commits-unsubscribe@users.apache.org[Unsubscribe]
+|http://mail-archives.apache.org/mod_mbox/opennlp-commits/[Archives]
```
Comment (kinow) [inline src/main/jbake/content/get-involved.ad]: And here.

File: src/main/jbake/content/get-involved.ad [hunk 4]
```diff
@@ -22,18 +22,47 @@
...
+
+|Issues Mailing List
+|This list follows changes and posts to JIRA, the projects bug tracking system.  If you have any
+issues please post them to the JIRA pages and not to this list. This list is mainly for developers to keep track of changes and issues to be resolved; however, any changes should be posted to the JIRA pages and _not_ _directly_ to this mailing list.
+|mailto:issues-subscribe@users.apache.org[Subscribe]
+|mailto:issues-unsubscribe@users.apache.org[Unsubscribe]
+|http://mail-archives.apache.org/mod_mbox/opennlp-issues/[Archives]
```
Comment (kinow) [inline src/main/jbake/content/get-involved.ad]: And this one too.

File: src/main/jbake/content/mailing-lists.ad [hunk 5]
```diff
@@ -22,52 +22,33 @@
...
-This list is for development discussions, patch suggestions, and current issues posted
+[cols="3,3,3,3,3"]
+|===
+|Mailing List |Purpose |Subscribe |Unsubscribe |Archives
+
+|Users Mailing List
+|This list is the general users mailing list and is for posting most, if not all, relevant questions. If you are trying to decide which list, opennlp-dev or opennlp-users, post here first. Someone will kindly suggest you post to opennlp-dev if it is more appropriate there.
+|mailto:users-subscribe@users.apache.org[Subscribe]
+|mailto:users-unsubscribe@users.apache.org[Unsubscribe]
+|http://mail-archives.apache.org/mod_mbox/opennlp-users/[Archives]
```
Comment (kinow) [inline src/main/jbake/content/mailing-lists.ad]: HTTPS

File: src/main/jbake/content/mailing-lists.ad [hunk 6]
```diff
@@ -22,52 +22,33 @@
...
-the JIRA pages and _not_ _directly_ to this mailing list.
-
-Subscribe to this list by sending an email to <issues-subscribe@opennlp.apache.org>
-
-To un-subscribe send an e-mail to <issues-unsubscribe@opennlp.apaache.org>
-
-http://mail-archives.apache.org/mod_mbox/opennlp-issues/[issues archive]
+|mailto:dev-subscribe@users.apache.org[Subscribe]
+|mailto:dev-unsubscribe@users.apache.org[Unsubscribe]
+|http://mail-archives.apache.org/mod_mbox/opennlp-dev/[Archives]
```
Comment (kinow) [inline src/main/jbake/content/mailing-lists.ad]: One more.

File: src/main/jbake/content/mailing-lists.ad [hunk 7]
```diff
@@ -22,52 +22,33 @@
...
+
+|Commits Mailing List
+|This list follows changes to the project. This list is _not_ for questions. This list is used by svn to post changes to the project. If you have questions about any content here, post them to the opennlp-dev list above.
+|mailto:commits-subscribe@users.apache.org[Subscribe]
+|mailto:commits-unsubscribe@users.apache.org[Unsubscribe]
+|http://mail-archives.apache.org/mod_mbox/opennlp-commits/[Archives]
```
Comment (kinow) [inline src/main/jbake/content/mailing-lists.ad]: Almost there.

File: src/main/jbake/content/mailing-lists.ad [hunk 8]
```diff
@@ -22,52 +22,33 @@
...
+
+|Issues Mailing List
+|This list follows changes and posts to JIRA, the projects bug tracking system.  If you have any
+issues please post them to the JIRA pages and not to this list. This list is mainly for developers to keep track of changes and issues to be resolved; however, any changes should be posted to the JIRA pages and _not_ _directly_ to this mailing list.
+|mailto:issues-subscribe@users.apache.org[Subscribe]
+|mailto:issues-unsubscribe@users.apache.org[Unsubscribe]
+|http://mail-archives.apache.org/mod_mbox/opennlp-issues/[Archives]
```
Comment (kinow) [inline src/main/jbake/content/mailing-lists.ad]: Last http link :D

**Comment by jzonthemtn** [comment]
Thanks @kinow!  See how those look.

**Review by kinow — APPROVED** [review]

**Pull request merged**